frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1097,12 +1117,4 @@ export const funnelLogic = kea<funnelLogicType>({
             }
         },
     }),
-
-    events: ({ values, actions }) => ({
```

Comment:
Decided to get rid of this! Wasn't solving the problem as I'd expect, will put up a new PR to solve this specific issue.